src/System.Net.Sockets/src/System/Net/Sockets/SendPacketsElement.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Contracts;
+
+namespace System.Net.Sockets
+{
+    // class that wraps the semantics of a winsock TRANSMIT_PACKETS_ELEMENTS struct
+    public class SendPacketsElement
+    {
+        internal string m_FilePath;
+        internal byte[] m_Buffer;
+        internal int m_Offset;
+        internal int m_Count;
+        internal Interop.Winsock.TransmitPacketsElementFlags m_Flags;
```

Comment:
This file looks like it's used for both Windows and Unix, but it has some Winsock-specific logic?